frontend/src/components/Modal/ConfirmationModal.tsx

```diff
@@ -25,18 +35,63 @@ const ConfirmationModal = ({ open, rider, onClose }: ConfirmationProps) => {
   };
 
   const studentDelete = () => {
```

Comment:
I think the function should be renamed. This Modal is now used to delete both students and employees. So, naming it only studentDelete is confusing. I think you should create two functions: `employeeDelete` and `studentDelete`, each one getting called conditionally.